False positive "Use of moved value" when Copy is derived by enumset

## Environment

* **IntelliJ Rust plugin version:** 0.3.136.3514-203
* **Rust toolchain version:** 1.48.0 (7eac88abb 2020-11-16) x86_64-unknown-linux-gnu
* **IDE name and version:** IntelliJ IDEA 2020.3 Community Edition (IC-203.5981.155)
* **Operating system:** Linux 5.8.16-2-manjaro
* **Macro expansion engine:** new
* **Name resolution engine:** new, old

## Problem description
I have an enum that derives Copy indirectly via the [enumset](https://docs.rs/enumset/1.0.1/) crate
```rust
#[derive(EnumSetType)] // also derives Copy, Clone, Eq, PartialEq
enum Test { A }
```
despite that, IDE still reports "Use of moved value" and suggests to derive Copy. However, doing so breaks compilation:
```
error[E0119]: conflicting implementations of trait `std::marker::Copy` for type `Test`:
 --> src/lib.rs:3:10
  |
3 | #[derive(EnumSetType, Clone, Copy)] // also derives Copy, Clone, Eq, PartialEq
  |          ^^^^^^^^^^^         ---- first implementation here
  |          |
  |          conflicting implementation for `Test`
  |
  = note: this error originates in a derive macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

## Steps to reproduce
Minimal example: https://gist.github.com/P4/45ff741b9300a8b1391062d40d471d24

--- a/src/main/resources-stable/META-INF/experiments.xml
+++ b/src/main/resources-stable/META-INF/experiments.xml
@@ -12,10 +12,10 @@
         <experimentalFeature id="org.rust.macros.proc" percentOfUsers="0">
             <description>Enables all types of procedural macro expansion</description>
         </experimentalFeature>
-        <experimentalFeature id="org.rust.macros.proc.function-like" percentOfUsers="0">
+        <experimentalFeature id="org.rust.macros.proc.function-like" percentOfUsers="100">
             <description>Enables function-like procedural macro expansion</description>
         </experimentalFeature>
-        <experimentalFeature id="org.rust.macros.proc.derive" percentOfUsers="0">
+        <experimentalFeature id="org.rust.macros.proc.derive" percentOfUsers="100">
             <description>Enables derive procedural macro expansion</description>
         </experimentalFeature>
         <experimentalFeature id="org.rust.macros.proc.attr" percentOfUsers="0">

